bin/prometheus-metadata-exporter

```diff
@@ -113,16 +113,27 @@ class OcspCheckMetric < Prometheus::Client::Gauge
   private
 
   def check_and_set(gauge, certificate_identities, ca_files)
+      return if certificate_identities.nil?
+
       pems = certificate_identities.keys
       ocsp_results = @pem_checker.check_pems(pems, ca_files)
       certificate_identities.map do |pem, identities|
         identities.map do |identity|
+          puts(identity)
           cert = @certificate_factory.from_inline_pem(pem)
+
+          ocsp_result = 1
+          if ocsp_results[pem].unknown?
+            ocsp_result = -1
```

Comment:
This isn't a very prometheus-style way of encoding information.  Right now we have a metric `verify_metadata_certificate_ocsp_success` which is `1` if ocsp says the certificate is valid and `0` if ocsp says the certificate is invalid.  Adding `-1` as a third state to the metric feels wrong.

Alternatives I would suggest:
1. don't emit a metric at all if OCSP information isn't present (ie skip the `gauge.set()` call below)
2. emit a separate metric which encodes whether OCSP information is present `verify_metadata_certificate_ocsp_known` where `1` means "known" and `0` means "unknown"

I'd probably go with 1 unless we have a need for this information.